web/libs/datamanager/src/components/MainView/GridView/GridView.jsx

```diff
@@ -56,11 +57,22 @@ const GridDataGroup = observer(({ type, value, field, row }) => {
 });
 
 const GridCell = observer(({ view, selected, row, fields, onClick, ...props }) => {
+  const { setCurrentTaskId, imageField } = useContext(GridViewContext);
+
+  const handleBodyClick = (e) => {
```

Comment:
I have seen more times that you have chosen not to wrap an inline function with useCallback than not, can you maybe explain the thinking? Generally speaking it is safer to default to always wrapping with useCallback to ensure there are no stale closures and that it is a proper dependency of any downstream components which are memoized, or as dependencies of other useCallback, useEffect or useMemo.

Reply:
the modal is just rendered only once for a task, so I didn't care, but you are right for sure, will update them in a sec